distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc

```diff
@@ -7,11 +7,21 @@
 
 The Tika Input Transformer is the default input transformer responsible for translating Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into Catalog Metacards.
```

Comment:
In the interest of standardization can we change "Catalog Metacards" to either "catalog entries" or "catalog records"?

Reply:
going with `catalog records`